examples/custom_potential/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.11)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules")
+
+find_package(READDY REQUIRED)
+
+message(STATUS "current source dir ${CMAKE_CURRENT_SOURCE_DIR}")
+message(STATUS "source dir ${CMAKE_SOURCE_DIR}")
+if("${CMAKE_CURRENT_SOURCE_DIR}" EQUAL "${CMAKE_SOURCE_DIR}")
+    message(STATUS "adding pybind subdir")
+    add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../../libraries/pybind11 ${CMAKE_CURRENT_BINARY_DIR}/pybind11)
+endif()
+
+add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../../libraries/pybind11 ${CMAKE_CURRENT_BINARY_DIR}/pybind11)
+
+set(SRC binding.cpp TestPotential.h TestPotential.cpp)
```

Comment:
does this statement `set(SRC ...)` do something? It seems redundant.

Reply:
oops you are right, this is some remainder of what i had before and now completely unnecessary :-)